skimage/filters/thresholding.py

```diff
@@ -881,3 +881,42 @@ def threshold_sauvola(image, window_size=15, k=0.2, r=None):
         r = 0.5 * (imax - imin)
     m, s = _mean_std(image, window_size)
     return m * (1 + k * ((s / r) - 1))
+
+
+def apply_hysteresis_threshold(image, low, high):
```

Comment:
No, we had a discussion (don't remember where) that `threshold_*` should always be a threshold mask. `threshold_sauvola` does indeed return a mask, not a binary image (the doctest example is in fact wrong, but the docstring description of the return value is correct). This is a completely different type of function from all the `threshold_*` functions.